libraries/rush-lib/src/utilities/AsyncRecycler.ts

```diff
@@ -179,4 +171,33 @@ export class AsyncRecycler {
     // The child won't stay alive unless we unlink it from the parent process
     process.unref();
   }
+
+  private _renameOrRecurseInFolder(folderPath: string): void {
+    const ordinal: number = this._movedFolderCount++;
+    const targetDir: string = path.join(this.recyclerFolder, `${this._prefix}_${ordinal}`);
```

Comment:
Do these have to have os-specific slashes, or can we just use forward slashes on Windows?

Reply:
Can user forward slashes.